app/assets/stylesheets/_application.scss

```diff
@@ -112,3 +112,8 @@
 
 // OpenID Connect (API)
 @import 'openid_connect_error_page';
+
+
+.btn-success {
+      box-shadow: 0px 1px #c2ef9b inset;
```

Comment:
- Color literals like `#c2ef9b` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Line should be indented 2 spaces, but was indented 6 spaces
- `0px` should be written without units as `0`